clusters/hive/pools/ocp-release-4.next-x86-64-for-4.13.0-0-to-4.14.0-0_clusterimageset.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterImageSet
+metadata:
+  annotations:
+    version_lower: 4.13.0-0
+    version_upper: 4.14.0-0
```

Comment:
until the clusterimageset updater understands the 4-dev-preview stream, i'm not sure what affect this is going to have.  Is there a way to prevent the clusterimageset updater from trying to touch this file, since it's presumably going to stomp the `releaseImage` value below?